### PR TITLE
add i18n to NumberQuestion

### PR DESCRIPTION
**Pull request by marcja**

### CODE REVIEW

File: client/src/components/NumberQuestion.jsx [hunk 1]
```diff
@@ -61,7 +61,8 @@ function NumberQuestion(props) {
         contains an `input` subcomponent, making the lint error inaccurate.
         eslint-disable-next-line jsx-a11y/label-has-associated-control */}
       <label htmlFor={a11yId}>{question.text}</label>
-      {!!question.hint && <p className="questionHint">{question.hint}</p>}
+      {!!question.hint &&
```
Comment (egracer) [inline client/src/components/NumberQuestion.jsx:64]: Will this conditionally hide the hint? I think even if `questionKey` is null, the `p` tag will always be rendered. I ran into this with `BooleanQuestion` and it actually looks ok even if the hint text is blank, so we could just remove the conditional rendering here altogether.

File: client/src/components/NumberQuestion.jsx [hunk 2]
```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {useState} from 'react';
 import {useSelector, useDispatch} from 'react-redux';
-import {selectQuestions} from '../features/questions/questionsSlice';
```
Comment (mscharfstein) [inline client/src/components/NumberQuestion.jsx:5]: we might not even need the questions in the redux store at all....

File: client/src/components/NumberQuestion.jsx (hunk 1 above)
Comment (marcja) [reply client/src/components/NumberQuestion.jsx:64]: It's your code, man. I just added the translations. ;)

File: client/src/components/NumberQuestion.jsx (hunk 2 above)
Comment (marcja) [reply client/src/components/NumberQuestion.jsx:5]: Still need to pull the `question.type`.

**Pull request merged**